externs/shaka/player.js

```diff
@@ -1397,7 +1398,14 @@ shaka.extern.StreamingConfiguration;
  *   SourceBuffer.changeType. RELOAD uses cycling of MediaSource.
  *   Defaults to SMOOTH if SMOOTH codec switching is supported, RELOAD
  *   overwise.
+ * @property {function(string): string} addExtraFeaturesToSourceBuffer
+ *   Callback to generate extra features striug based on used MIME type.
+ *   Some platforms may need to pass features when initializing the
+ *   sourceBuffer.
+ *   This string is ultimately appended to a MIME type in addSourceBuffer() &
+ *   changeType().
  * @property {string} sourceBufferExtraFeatures
```

Comment:
Since this change is going in the next major, we can remove this directly, please add a note in https://github.com/shaka-project/shaka-player/blob/main/docs/tutorials/upgrade.md?plain=1#L94

Reply:
OK, done!